doc/source/reference/maskedarray.generic.rst

```diff
@@ -496,4 +491,7 @@ Let's consider an array ``d`` of random floats between 0 and 1. We wish to
 compute the average of the values of ``d`` while ignoring any data outside
 the range ``[0.1, 0.9]``::
 
-   >>> print ma.masked_outside(d, 0.1, 0.9).mean()
+   >>> np.random.seed(42)
```

Comment:
Looks all good to me. I wonder what peoples opinion on this is. Should just add a `  # may vary` (not sure about the format) comment instead of using `np.random.seed` which we actually may want people to use less (while the replacement may be a bit long)?

In either case, I am fine with putting this in as is.

Reply:
I don't have any opinions on this, for the sake of the example should we just do a simple range to generate the input array? 🤷‍♀